tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -111,6 +111,10 @@ bool InternalCommandsHandler::executeWriteCommand(uint32_t requestSize,
                << " PRE_PROCESS_FLAG=" << ((flags & MsgFlag::PRE_PROCESS_FLAG) != 0 ? "true" : "false")
                << " HAS_PRE_PROCESSED_FLAG=" << ((flags & MsgFlag::HAS_PRE_PROCESSED_FLAG) != 0 ? "true" : "false"));
 
+  if (writeReq->header.type == WEDGE) {
+    LOG_INFO(m_logger, "A wedge command has been called in seqNum: " << KVLOG(sequenceNum));
```

Comment:
Nit: No need for "in seqNum". KVLOG will print the output as `sequenceNum: <ActualValue>`.

Reply:
fixed